platform.md

```diff
@@ -122,6 +125,13 @@ The buildpacks directory MUST contain unarchived buildpacks such that:
 - Each top-level directory is a buildpack ID.
 - Each second-level directory is a buildpack version.
 
+## OCI Image
+The OCI image containing the compiled application is generated by the Build Image during the [Build Phase](#build-phase).
```

Comment:
Image is not generated during the build phase -- it's generated during the export phase by the exporter.

Reply:
We've created an Export phase section in the document and linked to that.